auraescript/src/lib.rs

```diff
@@ -27,6 +27,63 @@
  *   limitations under the License.                                           *
  *                                                                            *
 \* -------------------------------------------------------------------------- */
+//! # AuraeScript
+//!
+//! AuraeScript is a turing complete language for platform teams built on [Deno](https://deno.land).
+//!
+//! AuraeScript is a lightweight client that wraps the [Aurae Standard Library](https://aurae.io/stdlib/).
+//!
+//! AuraeScript is a quick way to access the core Aurae APIs and follows normal UNIX parlance. AuraeScript should feel simple and intuitive for any Go, C, Python, or Rust programmer.
+//!
+//! ```typescript
```

Comment:
Please remove all of the typescript examples from this file. We should reference the files in `/examples` for examples of AuraeScript